main.go

```diff
@@ -81,13 +81,28 @@ var (
 		},
 		[]string{"remote"},
 	)
+	writeErrors = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "storage_bigquery_write_error_count",
```

Comment:
The other counter metrics end in `_total`, but this one ends in `_count`. For consistency I'd name this metric `storage_bigquery_write_error_total`

Reply:
Same here `errors` should be plural. I missed this in the first review.